Add docs about setting up permissions for GH apps & orgs (#8305)
readthedocs/readthedocs.org: docs/connected-accounts.rst

If you signed up or logged in to Read the Docs with your GitHub, Bitbucket, or GitLab
credentials, you're all done. Your account is connected.

> To connect a social account, go to your :guilabel:`Username dropdown` > :guilabel:`Settings` > :guilabel:`Connected Services`.

Reviewer: I think we should still mention the link, something like "go to this link or ...". It is easier to just click than search for the UI elements.

Author: We need a way to link to both sites, if we're providing a link..

    but there isn't a more granular permission
    that only allows setting up SSH keys for read access.

.. _github-permission-troubleshooting:

GitHub Permission Troubleshooting
`````````````````````````````````

A common issue with importing a GitHub project to Read the Docs is OAuth permissions.
Many organizations require approval for each OAuth application that is used,
or you might have disabled it in the past for your personal account.
This can happen at the personal or organization level,

> depending on where the project you are trying to access has permissions from.

Reviewer 2: It might be helpful to discuss the outcome to the user here as well. The question I always field is "why are my repositories not in my list of repositories I can import" or "why is my organization missing from the import page". I'd like to link to this section in the dashboard UI later, so hopefully can help direct users at the source.

Author: Aye -- I don't know the best way to structure it, so I just put it in bold for now.

    but there isn't a more granular permission
    that only allows setting up SSH keys for read access.

> .. _github-permission-troubleshooting:

Reviewer: We are using autolabel, there is no need for this, at least I'm missing something?

Author: I think I added this prior to adding the section label.

`````````````````````````````````

**Repositories not in your list to import**.

Many organizations require approval for each OAuth application that is used,
or you might have disabled it in the past for your personal account.
This can happen at the personal or organization level,
depending on where the project you are trying to access has permissions from.

.. tabs::

   .. tab:: Personal Account

       You need to make sure that you have granted access to the Read the Docs `OAuth App`_ to your **personal GitHub account**.
       If you do not see Read the Docs in the `OAuth App`_ settings, you might need to disconnect and reconnect the GitHub service.

       .. seealso:: GitHub docs on `requesting access to your personal OAuth`_ for step-by-step instructions.

       .. _OAuth App: https://github.com/settings/applications
       .. _requesting access to your personal OAuth: https://docs.github.com/en/organizations/restricting-access-to-your-organizations-data/approving-oauth-apps-for-your-organization

   .. tab:: Organization Account

       You need to make sure that you have granted access to the Read the Docs OAuth App to your **organization GitHub account**.

>        If you don't see |com_brand| listed, then you might need to connect GitHub in your social accounts as noted above.

Reviewer: Think this applies for both, .org and .com?

Author: Perhaps, is there a specific suggestion to improve it?

Author: Perhaps, is there a specific suggestion to improve it?

Reviewer 3: ```suggestion
       If you don't see "Read the Docs" listed, then you might need to connect GitHub to your social accounts as noted above.
```
